### ISSUE TITLE
Should we try to avoid long lines?

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/r-novice-gapminder#211]
On http://swcarpentry.github.io/r-novice-gapminder/05-data-structures-part2/, we have

```
df <- data.frame(id = c('a', 'b', 'c'), x = 1:3, y = c(TRUE, TRUE, FALSE), stringsAsFactors = FALSE)
```

Should we rewrite it as

```
df <- data.frame(id = c('a', 'b', 'c'),
    x = 1:3,
    y = c(TRUE, TRUE, FALSE),
    stringsAsFactors = FALSE)
```

to avoid long lines as well as make easy to learners see each one of the arguments?

**Comment by naupaka** [comment]
Yes, I think that would always be preferable.

**Comment by naupaka** [comment]
Fixed by #229
